src/sets.jl

```diff
@@ -197,11 +197,11 @@ end
 dimension(s::Union{ExponentialCone, DualExponentialCone, PowerCone, DualPowerCone}) = 3
 
 """
-    PositiveSemidefiniteConeTriangle(dimension)
+    PositiveSemidefiniteConeTriangle(side_dimension)
 
-The (vectorized) cone of symmetric positive semidefinite matrices, with side length `dimension` and with off-diagonals unscaled.
+The (vectorized) cone of symmetric positive semidefinite matrices, with side length `side_dimension` and with off-diagonals unscaled.
 The entries of the upper triangular part of the matrix are given column by column (or equivalently, the entries of the lower triangular part are given row by row).
-An ``dimension \\times dimension`` matrix has ``dimension(dimension+1)/2`` lower-triangular elements, so for the vectorized cone of dimension ``n``, the corresponding symmetric matrix has side dimension ``\\sqrt{1/4 + 2 n} - 1/2`` elements.
+A ``d \\times d`` matrix has ``d(d+1)/2`` lower-triangular elements, so for the vectorized cone of dimension ``n``, the corresponding symmetric matrix has a side dimension ``d`` of ``\\sqrt{1/4 + 2 n} - 1/2`` rows/columns, where ``d`` is the `side_dimension`.
```

Comment:
This sentence is still a bit awkward. Could you restructure it to first say what the point is ("here is how to calculate the side dimension from the flat dimension") and then provide the equations?

Reply:
I was using this as an opportunity to suggest a small fix to a pre-existing issue with the wording, so you're right that it's orthogonal to this change. Generally it's good to try and fix small things when you notice them up to the point where it becomes annoying. I'll tweak this sentence in your PR.